public/app/features/dashboard-scene/scene/Scopes/ScopesFiltersScene.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import { Scope } from '@grafana/data';
+import { SceneComponentProps, SceneObjectBase, SceneObjectState } from '@grafana/scenes';
+
+import { ScopesFiltersAdvancedSelectorScene } from './ScopesFiltersAdvancedSelectorScene';
+import { ScopesFiltersBasicSelectorScene } from './ScopesFiltersBasicSelectorScene';
+import { ScopesFiltersOpenAdvanced, ScopesFiltersSaveAdvanced } from './events';
+
+export interface ScopesFiltersSceneState extends SceneObjectState {
+  basicSelector: ScopesFiltersBasicSelectorScene;
+  advancedSelector: ScopesFiltersAdvancedSelectorScene;
+}
+
+export class ScopesFiltersScene extends SceneObjectBase<ScopesFiltersSceneState> {
+  static Component = ScopesFiltersSceneRenderer;
+
+  constructor() {
+    super({
+      basicSelector: new ScopesFiltersBasicSelectorScene(),
+      advancedSelector: new ScopesFiltersAdvancedSelectorScene(),
+    });
+
+    this.addActivationHandler(() => {
+      this.state.basicSelector.fetchBaseNodes();
+
+      const openAdvancedSubscription = this.subscribeToEvent(ScopesFiltersOpenAdvanced, ({ payload }) => {
+        this.state.advancedSelector.setState(payload);
+        this.state.advancedSelector.open();
+      });
+
+      const saveAdvancedSubscription = this.subscribeToEvent(ScopesFiltersSaveAdvanced, ({ payload }) => {
+        this.state.basicSelector.setState(payload);
+      });
+
+      return () => {
+        openAdvancedSubscription.unsubscribe();
+        saveAdvancedSubscription.unsubscribe();
+      };
```

Comment:
You don't really need to care about unsubscribing when using `subscribeToEvent`. `SceneObjectBase` internal deactivation handles the removal of subscriptions: https://github.com/grafana/scenes/blob/main/packages/scenes/src/core/SceneObjectBase.tsx#L311